release/_releases_json.sh

```diff
@@ -52,7 +52,19 @@ function _process_product {
 		tr -d "/" | \
 		uniq)
 	do
-		_process_product_version "${product_name}" "${product_version}"
+		local product_version_trivial=$(echo "${product_version}" | grep -oE '(\.u|-u)[0-9]+' | grep -oE '[0-9]+')
+
+		if [ "${product_version:2:1}" = 4 ]
```

Comment:
@pgydev, can you please explain what exactly is the purpose of `:2:1`?

Besides, can you please share an example of when the `product_version_trivial` will be empty?

Thanks

Reply:
@natocesarrego  When we use release like `2024.q3.3` it will be empty.
I gave a second thought for the :2:1 i adjust it.